activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+*   Add automatic filtering of encrypted attributes on inspect
+
+    This can be enabled by setting
+
+    ```ruby
+    config.active_record.encryption.add_to_filter_attributes = true
+    ```
```

Comment:
Sorry I missed this last time around!  This should be `add_to_filter_parameters` now, instead of `add_to_filter_attributes`.  And perhaps mention that it's enabled by default?

Reply:
Great catch!